app/tasks/bookmark_save_task.py

```
@@ -1,93 +1,231 @@
+            # star_id로 시작하는 모든 문서 찾아서 삭제
+            try:
+                # star_id를 포함하는 모든 source_id 패턴 삭제
+                patterns_to_delete = [
+                    f"{star_id}_chunk_",  # 청크들
+                    f"{star_id}_memo",    # 메모
+                    f"{star_id}_summary"  # 요약
+                ]
+                
+                for pattern in patterns_to_delete:
+                    # 패턴으로 시작하는 source_id들 삭제
+                    pattern_deleted = await vector_service.delete_documents_by_pattern(
```

Pylint (3.3.7)

[error] Instance of 'VectorService' has no 'delete_documents_by_pattern' member

(E1101)